modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -27,6 +27,15 @@
 		],
 		"to": "searchVocabularies(param#0#, new long[] {param#1#}, param#2#, new int[0], param#3#, param#4#, new Sort())"
 	},
+	{
+		"classNames": [
+			"JournalFolderLocalService",
+			"JournalFolderLocalServiceUtil"
+		],
+		"from": "addFolder(long, long, long, String, String, ServiceContext)",
+		"issueKey": "LPS-189928",
```

Comment:
@alberttgomes the LPS is wrong, should be LPS-184731. The ticket were the UpgradeJavaFolderParameterCheck was created